### PR TITLE
feat(suite-native): fiat rates hidden for testnet coins

### LINKED ISSUE #7210: Remove FIAT balance for Testnet coins
Remove FIAT rate / balance from testnet coin - they don't have any value.

### LINKED ISSUE #7591: During import of XPUB fiat value is not properly rendered in mobile app.
**Describe the bug**
During import of XPUB fiat value is not properly rendered in mobile app.

**Info:**

-   Suite Version: app/, b1b0ebf - [develop] - chore(mobile): enable iPad (#7583)

-   OS: Android 

**Screenshots**
image

### PR DESCRIPTION
**Pull request by PeKne**
## DEMO 
https://user-images.githubusercontent.com/26143964/215483042-c00d1f2c-dc0c-44e6-8834-1915ab328755.mov

closes #7210
fixes #7591

### CODE REVIEW

**Review by Nodonisko — CHANGES_REQUESTED** [review]
Wait until #7401 is merged

File: suite-common/wallet-core/src/accounts/accountsReducer.ts
Comment (juriczech) [inline suite-common/wallet-core/src/accounts/accountsReducer.ts:174]: You can do it like this I guess `isTestnet(account?.symbol) ?? false`. This might require allowing symbol param to be undefined, in which case the `isTestnet` function would return false which is okay I guess.

File: suite-native/accounts/src/components/AccountListItem.tsx
Comment (juriczech) [inline suite-native/accounts/src/components/AccountListItem.tsx]: I wouldn't render the discreet text at all if it's testnet. Discreet text renders two elements and might be better to not render it instead of passing it empty string

File: suite-native/assets/src/components/AssetItem.tsx
Comment (juriczech) [inline suite-native/assets/src/components/AssetItem.tsx]: Don't render discreet text if testnet

File: suite-native/module-accounts/src/components/AccountBalance.tsx
Comment (juriczech) [inline suite-native/module-accounts/src/components/AccountBalance.tsx]: same here

File: suite-native/transactions/src/components/TransactionDetail/TransactionDetailData.tsx
Comment (juriczech) [inline suite-native/transactions/src/components/TransactionDetail/TransactionDetailData.tsx]: Always prefix boolean values with is, has, should etc. So in this case it would be `isFiatVisible`.

Altough I  don't really like that you're using this condition inverted in some components, as it is in others and now you're using some vague name even though it's in just one place. 

Did you consider creating custom hook to determine whether account is testnet or not which would just return true / false based on symbol / accountKey and then using it in all of the components?

File: suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx
Comment (juriczech) [inline suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx]: no need for `?? 0` as fiatAmount won't be `undefined | null` in the formatter call

File: suite-native/transactions/src/components/TransactionDetail/TransactionDetailSheets.tsx
Comment (juriczech) [inline suite-native/transactions/src/components/TransactionDetail/TransactionDetailSheets.tsx]: `isValuesSheetVisible` would be a better name but consider a better way to handle it as mentioned in previous comments

**Comment by Nodonisko** [comment]
@PeKne Since #7401 You can simplify this PR a lot and just put simple `if` into `CryptoToFiatAmountFormatter`.

File: suite-native/accounts/src/components/AccountListItem.tsx
Comment (juriczech) [inline suite-native/accounts/src/components/AccountListItem.tsx:50]: Are you sure deleting this is not breaking anything?

File: suite-native/assets/src/components/AssetItem.tsx
Comment (juriczech) [inline suite-native/assets/src/components/AssetItem.tsx]: I would send value and symbol to formatter and handle logic there.

File: suite-native/transactions/src/components/TransactionDetail/TransactionDetailSheets.tsx
Comment (PeKne) [reply suite-native/transactions/src/components/TransactionDetail/TransactionDetailSheets.tsx]: fixed in 02dd1d1b5

File: suite-native/assets/src/components/AssetItem.tsx
Comment (PeKne) [reply suite-native/assets/src/components/AssetItem.tsx]: changed in ba28d5486

File: suite-native/accounts/src/components/AccountListItem.tsx
Comment (PeKne) [reply suite-native/accounts/src/components/AccountListItem.tsx:50]: Yes, it was tested.

File: suite-common/wallet-core/src/accounts/accountsReducer.ts
Comment (PeKne) [reply suite-common/wallet-core/src/accounts/accountsReducer.ts:174]: Unfortunately, i can't change it like that. `isTestnet` function does not accept `undefined`, it only takes one of the network symbols. And the `account?.symbol` is possibly undefined.

File: suite-native/atoms/src/Text.tsx
Comment (Nodonisko) [inline suite-native/atoms/src/Text.tsx]: why this?

**Review by Nodonisko — APPROVED** [review]

File: suite-native/atoms/src/Text.tsx
Comment (PeKne) [reply suite-native/atoms/src/Text.tsx]: I did it to make the Text component keep always the same height even if no `children` was provided. Empty Text component is used for example in the new `FiatAmountFormatter`:
```tsx
 if (!value || isTestnetValue) return <Text />;
 ```

Comment (Nodonisko) [reply suite-native/atoms/src/Text.tsx]: I would rather not do that here, because it could lead to unexpected behavior. Do this only in places of usage to keep this atoms as much pure as possible. So rather do `return <Text>{' '}</Text>`

**Review by juriczech — APPROVED** [review]

File: suite-native/atoms/src/Text.tsx
Comment (PeKne) [reply suite-native/atoms/src/Text.tsx]: fixed in c69893292

**Pull request merged**
